deploy/olm-catalog/operand-deployment-lifecycle-manager/1.2.1/operand-deployment-lifecycle-manager.v1.2.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,569 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - services/finalizers
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          - namespaces
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resourceNames:
+          - operand-deployment-lifecycle-manager
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+          - list
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          - deployments
+          verbs:
+          - get
+        - apiGroups:
+          - operator.ibm.com
+          - operators.coreos.com
+          resources:
+          - '*'
+          - operandregistries
+          - operandconfigs
+          - operandrequests
+          - operandbindinfos
+          verbs:
+          - '*'
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - nonResourceURLs:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: operand-deployment-lifecycle-manager
+      deployments:
+      - name: operand-deployment-lifecycle-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: operand-deployment-lifecycle-manager
+          strategy: {}
+          template:
+            metadata:
+              annotations:
+                productID: 068a62892a1e4db39641342e592daa25
+                productMetric: FREE
+                productName: IBM Cloud Platform Common Services
+                productVersion: 3.4.1
```

Comment:
The CS product version should be 3.4, right?

Reply:
I will change it back to `3.4.0`. I will ask Steve about which version should we use exactly.